docs/sources/guides/whats-new-in-v6-0.md

```diff
@@ -141,7 +141,7 @@ Read more about the short-lived token solution and how to configure it [here](/a
 
 > Please note that due to these changes, all users will be required to login upon next visit after upgrade.
 
-Besides these changes there have also been security improvements regarding protection against Cross-Site Request Forgery (CSRF) and Cross-site Scripting (XSS) attacks.
+Besides these changes we have also introduced [SameSite](https://www.owasp.org/index.php/SameSite) setting to protect against Cross-Site Request Forgery (CSRF) and Cross-site Scripting (XSS) attacks. This setting enables more control of when the browser include cookies in requests. Its set to `lax` by default but can be configured using `cookie_samesite` under `[security]`
```

Comment:
My idea with XSS protection here was to highlight the work we've been doing with disable script tags per default in the text panel. Maybe it was a bad idea to combine both new auth and other security fixes in same section?

Reply:
:+1: made some changes